client/Assets/Scripts/Game/UI/GemStoreController.cs

```diff
@@ -54,7 +58,7 @@ async void Start()
     // Update is called once per frame
     void Update()
     {
-        var desiredY = IsOpen ? 0 : -Screen.height*2;
+        var desiredY = IsOpen ? 0 : -Screen.height * 3f;
```

Comment:
Might be worth making this a local constant with a descriptive name. As it is, it isn't clear to me why we're multiplying screen height by a scalar. Also, have you tested the visual behavior of this on device? Probably totally fine as it is, but this is the sort of thing that can be subtly weird on different devices 😄